### DISCUSSION TITLE
Is it possible to enable work of other themes like clam and etc?

### DISCUSSION THREAD

**Discussion by arthurfait** [israel-dryer/ttkbootstrap#296] [Q&A]
Is it possible to enable work of other themes like clam and etc?

Or at least is it possible to add some method to prevent styles to be applied on startup?

I made somem tests and seems its possible.

Please see https://github.com/arthurfait/ttkbootstrap/commit/3b1090473e7ef691fccc09e1af54e7298bec8b80
(BTW in many places there are  spaces in the EOL.

**Comment by israel-dryer** [comment]
It's possible, but there are a lot of accommodations that would need to be made to make it work with multiple themes, especially for image-based widget styles. We'd need to extract the colors from those existing themes and make them work (for example the meter widget). Also, not all ttk themes share the same style properties. Clam is the most flexible of the styles, which is why I chose it as a base. The new default windows theme for windows 11 is actually very beautiful out-of-the-box, but it cannot be altered. If you want to create a pull request and are willing to work through some of those issues, I don't mind assisting.
